### PR TITLE
Add TargetFramework : netstandard1.3

### PR DESCRIPTION
**Pull request by StefH**
Added target framework **netstandard 1.3** to make it possible to use this library in netstandard 1.3 libraries.

### CODE REVIEW

File: src/jmespath.net/jmespath.net.csproj [hunk 1]
```diff
@@ -1,10 +1,10 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <Description>JmesPath.Net is a fully compliant JMESPATH parser.</Description>
     <AssemblyTitle>JmesPath.Net</AssemblyTitle>
     <Authors>Jeremie Devillard;Maxime Labelle</Authors>
-    <TargetFrameworks>netstandard2.0;net45</TargetFrameworks>
+    <TargetFrameworks>netstandard1.3;netstandard2.0;net45</TargetFrameworks>
```
Comment (springcomp) [inline src/jmespath.net/jmespath.net.csproj]: Please, update `src/jmespath.net.signed/jmespath.net.signed.csproj` accordingly.

Comment (springcomp) [reply src/jmespath.net/jmespath.net.csproj]: Please, update the corresponding `tests/jmespathnet.tests/jmespathnet.tests.csproj` with an additionnal target framework defined to `netcoreapp1.0`.

Also, add a corresponding trigger for this test configuration in `appveyor.yml` by adding a line in the `test_script` section of the file.

**Comment by springcomp** [comment]
With the requested changes, this pull request would be candidate for integration after #12.
Please, review the #12 pull request as a guideline for the changes.

**Comment by StefH** [comment]
I've added the new framework also to the signed project.
And updated the unit-test project.

Not need to update the appveyor for the unit-tests, both frameworks are tested. (See the appveyor build output)

**Pull request merged**